mod_reforged/hooks/items/weapons/greenskins/goblin_crossbow.nut

```diff
@@ -12,10 +12,7 @@
 
 		this.addSkill(::new("scripts/skills/actives/shoot_bolt"));
 
-		local reload = ::MSU.new("scripts/skills/actives/reload_bolt", function(o) {
-			o.m.ActionPointCost += 1;
-			o.m.FatigueCost += 5;
-		});
+		local reload = ::MSU.new("scripts/skills/actives/reload_bolt");
```

Comment:
We did decide to go with increased AP cost on windlass type crossbows, but which is then reduced to normal by Crossbow Mastery. So please leave the ActionPoint += 1 in.